CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @andresilva @gavofyork @tomusdrw
```

Comment:
How did you pick the CODEOWNERS here?
I usually looked at the history of the original files; in this case it would be only Gav, and then you'll have to plug the second slot with either you or me.

Reply:
Hmm okay. Guess I picked it in the wrong way. Will fix.